### PR TITLE
Repository interfaces

### PR DESCRIPTION
**Pull request by ranjeet1515**
1. Added all repository interfaces

---
This change is [Reviewable](https://reviewable.io/reviews/yugandhargangu/bettle-track/18)

### CODE REVIEW

**Comment by yugandhargangu** [comment]
Reviewed 59 of 59 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/yugandhargangu/bettle-track/18)*

**Review by yugandhargangu — APPROVED** [review]

File: application/bettle/bettle-repo/src/test/resources/test.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+spring.datasource.url=jdbc:h2:mem:AZ;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
```
Comment (shashankitmaster) [inline application/bettle/bettle-repo/src/test/resources/test.properties:1]: You should write test.yml instead of `test.properties`

**Review by shashankitmaster — CHANGES_REQUESTED** [review]

File: application/bettle/bettle-repo/src/test/resources/test.properties (hunk 1 above)
Comment (ranjeet1515) [reply application/bettle/bettle-repo/src/test/resources/test.properties:1]: we will update it in to application/yml branch

**Comment by shashankitmaster** [comment]
Reviewed 59 of 59 files at r1.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/yugandhargangu/bettle-track/18)*

**Review by shashankitmaster — APPROVED** [review]

**Pull request merged**